runtime/doc/options.txt

```diff
@@ -3991,6 +3992,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	"precedes".  "SpecialKey" for "nbsp", "space", "tab" and "trail".
 	|hl-NonText| |hl-SpecialKey|
 
+						*'incsubstitute'* *'ics'*
+'incsubstitute' 'ics'	string	(default "")
+			global
+			
+	If set to "split" or "nosplit", substitutions (|:s|) are updated live
+	while the user types the command.  If set to "split", a split window
+	is open which displays the lines where the search matches. The
+	replacement text in the split is hightlighted using
+	|hl-IncSubstitute|. Note: Only '/' is supported as a delimiter.
```

Comment:
AFAICT all delimiters are supported now. 
Also this help entry should be located next to 'incsearch'.

I'll update this when merging.